#Datos para la deteccion de objetos
float32[] Distancia
float32[] Angulo
string[] Clase

#Datos para la deteccion de carriles
float32[] Puntos300
float32[] Puntos400

#Distancia minima detectada por el laser
float32 dist_laser
--- 
#Respuestas
